src/common/playlist/playlist-manager.js

```diff
@@ -280,8 +280,12 @@ class PlaylistManager {
     PlaylistManager._logger.debug(`Playing item number ${index}`, activeItem);
     const playback: Object = {loop: false};
     if (this._playlist.current) {
-      // from the second item onwards
-      playback['autoplay'] = true;
+      if (this._player._playbackStart) {
```

Comment:
did you verify with product that this is the intended behavior ?

Reply:
yes, david said that as long the functionality inside the playlist not changed and change entry will start play automatically he approve the change. this change only affect if user did not click first play on playlist. after first play move to other entry will play automatically.